src/components/CaVsNy.js

```diff
@@ -5,70 +5,64 @@ import axios from "axios";
 // Visualization for both CA and NY
 const CaVsNy = () => {
     const [chartData, setChartData] = useState({});
-    
+
     const chart = () => {
-      let filteredCA, filteredNy= [];  
+      let filteredCA, filteredNy= [];
       let covidDates = [];
       let covidCaDeath = [];
       let covidNyDeath = [];
 
       axios
         .get("https://covidtracking.com/api/v1/states/daily.json")
         .then(res => {
-            const state = res.data 
+            const state = res.data
             filteredCA = state.filter(a => a.state===("CA"))
-            
-
-            for (const dataObj of filteredCA){
+            for (const dataObj of filteredCA) {
                 covidDates.push(parseInt(dataObj.date));
                 covidCaDeath.push(dataObj.death);
             }
 
-            const ny = res.data
-            filteredNy = ny.filter(b => b.state===("NY"))
+            // TODO: use more verbose naming, e.g. newYorkData
+            filteredNy = res.data.filter(b => b.state===("NY"))
 
             for (const dataObj of filteredNy) {
                 covidNyDeath.push(dataObj.death)
             }
 
-            // CA key and value pair matching to support sorting the dates in ascending order 
-            var i; 
+            // CA key and value pair matching to support sorting the dates in ascending order
+            var i;
             var currentKey;
-            var currentVal; 
+            var currentVal;
             var masterList = {};
             for (i = 0; i < covidDates.length; i++){
                 currentKey = covidDates[i]
                 currentVal = covidCaDeath[i]
-                masterList[currentKey] = currentVal; 
+                masterList[currentKey] = currentVal;
             }
             console.log(masterList)
 
-            //for NY 
-            var j; 
```

Comment:
It's best to always use `let` or `const`. If you're unfamiliar with immutability, would consider reading the following:

https://medium.com/dailyjs/the-state-of-immutability-169d2cd11310
https://medium.com/@cristiansalcescu/learn-immutability-with-javascript-6a67e4a48d7f